dev-support/create-release/do-release-docker.sh

```diff
@@ -71,6 +72,7 @@ Options:
   -j [path]    path to local JDK installation to use building. By default the script will
                use openjdk8 installed in the docker image.
   -p [project] project to build, such as 'hbase' or 'hbase-thirdparty'; defaults to $PROJECT env var
+  -r [repo]    git repo to use for remote git operations. defaults to ASF gitbox for project.
```

Comment:
Why this? To save on time?

Reply:
yeah. on OS X any file operation we can avoid in a bind mount will help save us time. This skips both the network cost of cloning the remote repo (which is pretty big currently) and also skips local file copies.